docs/2_3_common_states.adoc

```diff
@@ -651,8 +651,8 @@ This argument is only valid in Model Exchange.
 
 [[nextEventTime,`nextEventTime`]]
 * The return argument <<nextEventTime>> contains the absolute time of the next time event latexmath:[T_{\mathit{next}}] if <<nextEventTimeDefined,`nextEventTimeDefined = fmi3True`>>.
-The importer must compute up to <<nextEventTime>> (or if needed slightly further) and then enter <<EventMode>> using <<fmi3EnterEventMode>> with argument <<timeEvent,`timeEvent = fmi3True`>>.
-The FMU must handle this time event at the first call to <<fmi3EnterEventMode>> with <<timeEvent,`timeEvent = fmi3True`>> at or after <<nextEventTime>>. +
+The importer must compute up to <<nextEventTime>> (or if needed slightly further) and then enter <<EventMode>> using <<fmi3EnterEventMode>>.
+The FMU must handle this time event at the first call to <<fmi3EnterEventMode>> at or after <<nextEventTime>>. +
```

Comment:
> The FMU must handle this time event at the first call to` <<fmi3EnterEventMode>>` at or after `<<nextEventTime>>.`

First part not exactly true, second don't make sense, (probably leftovers from rewrites)

I think we can just remove the sentence completely, it sufficient to require the importer to enter event mode.

Reply:
I think this sentence is important to explain how slight time computation differences can be resolved ("at or after").

Reply:
Sorry for delayed response

The FMU must handle this time event at the first call to <<fmi3EnterEventMode>> at or after <<nextEventTime>>.
Is not correct this sentence hints towards that the actual call enterEventMode will resolve the time event. This is not correct but rather the event iteration. As this can be interpreted as is stated by some person and such a person might later demand that it is not corrected in a bug release as it would be a backwards incompatible change we need to fix this sentence now.

Proposal:

The FMU must handle this time event during the event mode that is entered in by first call to <<fmi3EnterEventMode>> at or after <<nextEventTime>>.


I will make a pull request

Reply:
Sorry for delayed response

The FMU must handle this time event at the first call to <<fmi3EnterEventMode>> at or after <<nextEventTime>>.
Is not correct this sentence hints towards that the actual call enterEventMode will resolve the time event. This is not correct but rather the event iteration. As this can be interpreted as is stated by some person and such a person might later demand that it is not corrected in a bug release as it would be a backwards incompatible change we need to fix this sentence now.

Proposal:

The FMU must handle this time event during the event mode that is entered in by first call to <<fmi3EnterEventMode>> at or after <<nextEventTime>>.


I will make a pull request